servers/rendering/renderer_viewport.cpp

```diff
@@ -1078,7 +1082,11 @@ RID RendererViewport::viewport_get_texture(RID p_viewport) const {
 	const Viewport *viewport = viewport_owner.get_or_null(p_viewport);
 	ERR_FAIL_NULL_V(viewport, RID());
 
-	return RSG::texture_storage->render_target_get_texture(viewport->render_target);
+	if (viewport->viewport_mode == RS::VIEWPORT_MODE_2D_AND_3D || viewport->render_buffers.is_null()) {
```

Comment:
So I'm guessing this should return our render target also when `viewport_mode == RS::VIEWPORT_MODE_XR && view_count = 1`

Reply:
I meant to ask you about that. I have no idea what the expected return value should be when using the XR mode. 

From this comment it sounds like:
1. If view_count is 1 return the render_target
2. if view_count is greater then one then return something else?